zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/GroupRequestValidator.java

```diff
@@ -60,7 +60,7 @@ public static void validateMemberId(
         validateId(entityId, "mappingId", violations);
         break;
       default:
-        validateId(entityId, "memberId", violations);
+        validateId(entityId, "entityId", violations);
```

Comment:
@nicpuppa I think there are more places where your IDE unintentionally changed this naming. I dind't highlight them all 😄